### ISSUE TITLE
Format code as monospace (and highlighted) in editor window

### ISSUE THREAD

**Issue by thesecretmaster** [Ionaru/easy-markdown-editor#96]
**Is your feature request related to a problem? Please describe.**
When typing code into the editor, it's always frustrating that it's not monospace and doesn't have syntax highlighting. Syntax highlighting is less important than it being monospace, but both are annoying.

**Describe the solution you'd like**
 It'd be really useful if when a line was indented by four spaces or surrounded by fences, it could be converted to a monospace font and possibly also get some kind of syntax highlighting.

**Describe alternatives you've considered**
I looked for ways to add my own CSS to do this, but I really can't seem to understand how you manage to style text inside a textarea. I'd be happy to make a PR for this if you'd point me to the right place to put my CSS.

**Comment by Ionaru** [comment]
You can style `cm-comment` to get monospace working.

```css
.cm-comment {
    font-family: monospace;
}
```

This isn't perfect, however. As spaces inside code blocks are not monospaced. As far as I can see there's no (easy) way to get that working in this version of CodeMirror.

**Comment by stephencmorton** [comment]
Yes, it would be really useful for programmers or anybody who frequently puts code blocks in their markdown.
